tsconfig.json

```diff
@@ -17,7 +17,9 @@
         "noImplicitAny": true,
         "strictFunctionTypes": true,
         "strictNullChecks": true,
-        "noImplicitThis": true
+        "noImplicitThis": true,
+        "esModuleInterop": true,
```

Comment:
Setting [esModuleInterop](https://www.typescriptlang.org/tsconfig#esModuleInterop) to true is [recommended](https://github.com/microsoft/TypeScript/issues/41139) and required for [compatibility](https://github.com/kubernetes-client/javascript/issues/751) with later versions of the K8s client library.